docs/functional-php.md

```diff
@@ -266,6 +267,28 @@ sort($collection, function($user1, $user2) {
 });
 ```
 
+## sorty_by()
+Sorts a collection by a value of some function
```

Comment:
I'd add "optionally preserving the array keys" to match with `sort()` above.

Reply:
@phanan Fixed